Makefile

```diff
@@ -267,6 +268,7 @@ scripts_install : $(SCRIPTS_INSTALL_DIR) \
 	install_storage_service_script \
 	install_north_script \
 	install_north_c_script \
+	install_notification_c_script \
```

Comment:
@MarkRiddoch I have checked in the code for `notification_c` script.
If notification service is pluggable thing then this should be the part of **foglamp-service-notification** repo as _services/foglamp.services.notification_ binary is being copied from there to FOGLAMP_ROOT. Please confirm on the installation part.

Reply:
The notification service is a separate and optional package, so perhaps the is better in that package, but we need the API to start it in FogLAMP itself, so it is not unreasonable to have the script  here.